components/02-elements/list/_list.scss

```diff
@@ -43,6 +46,31 @@ $list-divider: 1px solid $gray-dark !default;
             padding: $list-padding--horizontal;
         }
     }
+    &--desktop-hotizontal {
+        @include mq($screen-m) {
+            display: flex;
+            flex-direction: row;
+            flex-wrap: nowrap;
+        }
+        .list__item {
+            border-bottom: $list-divider;
+            padding: $list-padding--horizontal;
+            &:first-child {
+                @include mq($screen-m) {
+                    border-top: 0;
+                }
+            }
+            &:last-child {
+                border-bottom: none;
+            }
+            @include mq($screen-m) {
+                border-bottom: 0;
+            }
+        }
+        .list__link {
+            display: block;
+        }
+    }
```

Comment:
I already merged develop here, b/c of conflicts, so just update your code in the theme, to use new modificators.